pkg/worker/worker.go

```diff
@@ -76,6 +78,63 @@ type ContainerInstance struct {
 	Port         int
 	OutputWriter *common.OutputWriter
 	LogBuffer    *common.LogBuffer
+
+	memoryEvents MemoryEvents
+}
+
+func (i *ContainerInstance) isOOMKilled() bool {
```

Comment:
if this is on container instance - maybe let's move container instance to another file

Reply:
Moved to the lifecycle.go file.